symfony/messenger/4.3/post-install.txt

```diff
@@ -0,0 +1,18 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * You're ready to use the Messenger component. You can define your own message buses
+    or start using the default one right now by injecting the <info>messenger.bus.default</info> service
+    or typehinting <info>Symfony\Component\Messenger\MessageBusInterface</info> in your code.
+
+  * If you need to send messages to your broker, you can benefit from the built-in
+    To send messages to a transport and handle them asynchronously:
+
+    1. Uncomment the <comment>MESSENGER_TRANSPORT_DSN</> env var in .env
+       and <comment>framework.messenger.transports.async</> in config/packages/messenger.yaml;
+    2. Routing your messages to the async sender.
+
+  * If you need to send and handle your messages, uncomment the sync transport and add it on your routing
```

Comment:
I think we don’t need this. Messages are “sync” by default - you don’t need to do anything to make them sync. The sync transport is just handy if you want to be explicit, or maybe send a message to an async transport AND also handle it immediately.